### ISSUE TITLE
OUI-3371 | Tab don't get the scroll element when there first input is an empty array.

### ISSUE THREAD

**Issue by ice-blaze** [oblique-bit/oblique#150]
### Oblique version

12

### Is this a regression?

Yes

### Screenshots

![image](https://github.com/user-attachments/assets/3efaf170-b12b-401c-9c65-22ddf2b8dab2)

### Description

```true
Tab scroll doesn't appear if tab are initialized with empty array.
```

### Reproduction

```true
Initialise tabs with empty array.
Then setTimeout and add few tabs.
Ensure you have a lot of tabs, larger than the width of the webpage.
https://stackblitz.com/edit/oblique-master-layout-yybkum?file=src%2Fapp%2Fapp.component.ts
```

### Expected behaviour

```true
We should see two arrow buttons on the side of the tabs bar.
```

### Context

```true
One solution, set the a dumb tab first.
```

**Comment by OlivierPerceboisGarve** [comment]
Hi Ice-blaze,

we will try to repro it with that new information. I have updated the existing ticket : OUI-3371

**Comment by nina-egger** [comment]
This has been fixed with the version 12.2.1

**Comment by ice-blaze** [comment]
Confirmf it works now 👍
